app/res/values/strings.xml

```diff
@@ -153,6 +153,11 @@
     <string name="permission_commcare_logout_label">Allow Auto CommCare Logout</string>
     <string name="permission_commcare_logout_description">Allows this app to logout a user from CommmCare App</string>
 
+    <string name="permission_protect_falsely_exposed_components_label">Protect falsely exposed components</string>
+    <string name="permission_protect_falsely_exposed_components_description">Due to implicit intent constrains, certain intent-aware components are now exposed, this permission is to prevent abusive use by external apps</string>
+
+    <string name="permission_group_internal_action_label">Permission group for CommCare Internal Actions</string>
```

Comment:
can we try to simply these strings and make them user-friendly (remove any tech jargain like intents/callouts) as these may be presented to the users.

Reply:
Done [abe2a90](https://github.com/dimagi/commcare-android/pull/2930/commits/abe2a902dfc8701c9d6e2a61c74fc1203cae328f)